app/src/main/java/com/example/annotations/MainActivity.kt

```diff
@@ -57,5 +59,32 @@ class MainActivity : AppCompatActivity() {
             text = spannableString
             movementMethod = LinkMovementMethod.getInstance()
         }
+
+        configureMultipleClickTextView()
+    }
+
+    companion object {
+        const val androidStudioTermsAndConditions = "https://developer.android.com/studio/terms"
+        const val googlePrivacyAndPolicy = "https://policies.google.com/privacy?hl=en-US"
+    }
+
+    private fun configureMultipleClickTextView() {
+        val termsAndConditionsKey = "terms_and_conditions"
+        val privacyAndPolicyKey = "privacy_and_policy"
+
+        multipleClicksTextView.setClickListenerToText(this,
+            R.string.terms_and_conditions,
+            listOf(termsAndConditionsKey, privacyAndPolicyKey)
+        ) { selectedText ->
+            when (selectedText) {
```

Comment:
matching of annotation to action is kinda done twice: one is here and another one in `TextViewExtensions` https://github.com/iwuvhugs/Annotations/pull/3/files#diff-65dae317537b665779c105fee9b5d361R59.

Instead there could be created a simple data class that encapsulates annotation and action that needs to be invoked on click, ex:
```
data class AnnotationAndAction(val annotation: String, val action: (() -> Unit))
```
or possibly use Pairs or any other Kotlin niceties